actions/build_deploy_python_executable/action.yml

```diff
@@ -62,6 +62,7 @@ runs:
       run: >
```

Comment:
i think in other contexts we use this input to be the base deployment to use if its a PR, and the actual deployment name to be used if its a full deployment. I have mixed feelings about that but we should probably be consistent? i.e. can we get away without needing a new input here?

See for example https://github.com/dagster-io/dagster-cloud-action/blob/main/actions/utils/ci-init/action.yml#L11-L14 from the 'new' github action flow

If we do add an input we should document it here

Reply:
Updated this action to re-use the same input. The others it's a little tricky to, so I left the new input.

I can't say I have a great understanding of which actions are in use right now - at some point we can deprecate the non `ci` ones, right?